spec/controllers/data_controller_spec.rb

```diff
@@ -1,5 +1,12 @@
 require 'rails_helper'
 
 RSpec.describe DataController, :type => :controller do
+  it "provides bike data" do
+    session[:bikes] =[ { "name" => "Bike", "distance" => "100000" }]
```

Comment:
Surrounding space missing for operator '='.<br>Space inside square brackets detected.